docs/learn-balance-transfers.md

```diff
@@ -5,26 +5,50 @@ sidebar_label: How to transfer Balances
 ---
 
 Balance transfers are used to send one balance from one account to another account. To start
-transferring balances, we will begin by using the Polkadot-JS User Interface.
+transferring balances, we will begin by using [Polkadot-JS Apps][]. This guide assumes that you've
+already [created a Polkadot or Kusama account](learn-account-generation) and have some funds that
```

Comment:
I would say "already created an account" rather than specify Polkadot or Kusama, as this would let us use this page on other Substrate chains as well - Kulupu, Subsocial, etc.

Reply:
```suggestion
already [created an account](learn-account-generation) and have some funds that
```